proto/hts/participant/service.proto

```diff
@@ -57,6 +58,23 @@ message GetUpcomingEventsRequest {
     google.protobuf.Timestamp end = 2;
 }
 
+message GetEventsByTagIdRequest {
+    int64 tag_id = 1;
+}
+message GetEventsByOrganizationIdRequest {
+    int64 organization_id = 1;
+}
+
+message GetEventsByFacilityIdRequest {
+    int64 facility_id = 1;
```

Comment:
can merge all 3 messages to int64 id = 1

Reply:
what do you mean?

Reply:
GetByFacility, GetByEvent, ... have same type of input and can be generalized into one message